examples/chart/index.html

```diff
@@ -0,0 +1,19 @@
+<html>
+  <head>
+    <title>Teleport Helm chart repo</title>
+  </head>
+  <body>
+    <h1>Teleport Helm chart repo</h1>
+    <h3>Add the repo</h3>
+    <pre>helm repo add teleport https://charts.releases.teleport.dev</pre>
+    <h3>Install a chart</h3>
+    <pre><b><a href="https://github.com/gravitational/teleport/tree/master/examples/chart/teleport">teleport:</a></b> helm install [release-name] teleport/teleport</pre>
+    <pre><b><a href="https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-kube-agent">teleport-kube-agent:</a></b>
+    helm install [release-name] teleport/teleport-kube-agent \
+    --create-namespace \
+    --namespace [namespace] \
```

Comment:
saw a clever trick in a [blog post](https://www.vidarholen.net/contents/blog/?p=958) recently: use `${NAMESPACE?}` in command placeholders.
if a user executes the command without updating placeholders, it will fail because env vars are undefined.

Reply:
I like it!